src/org/rascalmpl/debug/IRascalMonitor.java

```diff
@@ -31,17 +32,28 @@ default void jobStart(String name, int totalWork) {
 		jobStart(name, 1, totalWork);
 	}
 
-	default void job(String name, Supplier<Boolean> block) {
+	default void job(String name, int totalWork, Supplier<Boolean> block) {
```

Comment:
I stand corrected; indeed I added this parameter. I think it's better we think about `totalWork` everywhere we call this interface and let the compiler tell us where we did not do that yet. Otherwise the UI will be quite un-intuitive.